src/layouts/homepage.html.jade

```diff
@@ -34,19 +34,20 @@ html(lang="en")
                         span.title Cozy&nbsp;
                         | a Personal Cloud
                         br
-                        | You can Host, Hack
-                        | and Delete
-                    h4
-                        | Your contacts, calendars and files are stored on your
-                        | hardware and synced with your devices
+                        | you can host, customize
+                        | and fully control
```

Comment:
After reading your explanation @Bobbylamouche, I must agree with you. While it may or may not be the best term, it's still better than what we currently have :)